website/views/shared/profiles/stats.jade

```diff
@@ -28,7 +28,8 @@ unless mobile
     tr(ng-repeat='(k,v) in profile.items.gear.equipped', ng-init='piece=Content.gear.flat[v]', ng-show='piece')
       td
         strong {{piece.text()}}:&nbsp;
-        span(ng-repeat='stat in ["str","con","per","int"]', ng-show='piece[stat]') {{piece[stat]}} {{stat.toUpperCase()}}&nbsp;
+        span(ng-repeat='stat in ["str","con","per","int"]', ng-show='piece[stat]') {{piece[stat]}} {{env.t(stat)}}&nbsp;
+        span(ng-hide='piece.str || piece.con || piece.per || piece.int')=env.t('noBenefit')
```

Comment:
The only reason I can think of that you'd want to submit an issue first would be to save you any code work if it was decided that the thing you opened an issue about did not need to be fixed, or needed to be approached in a different way than you were anticipating.

So, I'm fine with a PR. Makes it easier for me to review and think about to see the actual code, and not just the proposed code. :smile: